tests/tests.py

```diff
@@ -6008,6 +5937,51 @@ def test_plantcv_threshold_texture():
     else:
         assert 0
 
+def create_test_img(sz_img):
+    img = np.random.randint(np.prod(sz_img), size=sz_img) * 255
+    img = img.astype(np.uint8)
+    return img
+
+def create_test_img_bin(sz_img):
+    img = np.zeros(sz_img)
+    img[3:7, 2:8] = 1
+    return img
+@pytest.mark.parametrize("img, refimg, method, refimg_, pts, refpts, debug_mode", [
```

Comment:
do we need to test that the debug images work as expected? there is an above average amount of logic in that code. here we at least make sure it runs without an error but that doesn't insure it runs correctly....